src/index.spec.ts

```diff
@@ -192,6 +192,116 @@ describe('SDK', () => {
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-boolean element inputFloatingLabel', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { inputFloatingLabel: 'test' })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('inputFloatingLabel option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-string element buttonBackgroundColor', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { buttonBackgroundColor: 4 })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('buttonBackgroundColor option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-number element buttonBorderRadius', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { buttonBorderRadius: 'test' })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('buttonBorderRadius option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-boolean element buttonBoxShadow', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { buttonBoxShadow: 'test' })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('buttonBoxShadow option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-string element fontFamily', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { fontFamily: 4 })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('fontFamily option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
+  test('displays error when it receives a non-number element baseFontSize', () => {
+    const host = document.createElement('div')
+    host.setAttribute('id', 'easypay-checkout')
+    document.body.appendChild(host)
+    // @ts-ignore
+    startCheckout(manifest, { baseFontSize: 'test' })
+    expect(consoleSpy).toHaveBeenCalledWith(expect.stringContaining('baseFontSize option'))
+    expect(document.querySelector('#easypay-checkout iframe')).toBeNull()
+  })
+
```

Comment:
We should have a test or two to verify that it passes the properties to the iframe when given as parameters.

Reply:
Will do.